modules/localstack/src/main/java/org/testcontainers/containers/localstack/LocalStackContainer.java

```diff
@@ -124,10 +156,74 @@ public LocalStackContainer withServices(Service... services) {
             .withCredentials(localstack.getDefaultCredentialsProvider())
             .build();
      </code></pre>
+     * or for AWS SDK v2 you can use {@link #getDefaultAccessKey()}, {@link #getDefaultSecretKey()} directly:
+     * <pre><code>S3Client s3 = S3Client
+             .builder()
+             .endpointOverride(localstack.getEndpointOverride(LocalStackContainer.Service.S3))
+             .credentialsProvider(StaticCredentialsProvider.create(AwsBasicCredentials.create(
+             localstack.getDefaultAccessKey(), localstack.getDefaultSecretKey()
+             )))
+             .region(Region.of(localstack.getDefaultRegion()))
+             .build()
+     </code></pre>
      * @return an {@link AWSCredentialsProvider}
      */
     public AWSCredentialsProvider getDefaultCredentialsProvider() {
-        return new AWSStaticCredentialsProvider(new BasicAWSCredentials("accesskey", "secretkey"));
+        return new AWSStaticCredentialsProvider(new BasicAWSCredentials(getDefaultAccessKey(), getDefaultSecretKey()));
+    }
+
+    /**
+     * Provides a default access key that is preconfigured to communicate with a given simulated service.
+     * The access key can be used to construct AWS SDK v2 clients:
+     * <pre><code>S3Client s3 = S3Client
+             .builder()
+             .endpointOverride(localstack.getEndpointOverride(LocalStackContainer.Service.S3))
+             .credentialsProvider(StaticCredentialsProvider.create(AwsBasicCredentials.create(
+             localstack.getDefaultAccessKey(), localstack.getDefaultSecretKey()
+             )))
+             .region(Region.of(localstack.getDefaultRegion()))
+             .build()
+     </code></pre>
+     * @return a default access key
+     */
+    public String getDefaultAccessKey() {
```

Comment:
perhaps we should use just `getAccessKey` here and in other methods

Reply:
fixed in https://github.com/testcontainers/testcontainers-java/pull/2579/commits/2ff3fa12b78ec8dbca2bebb03168ab1e95ccf4d3